docs/technical-overview/compatibility.md

```diff
@@ -0,0 +1,22 @@
+# Compatibility
+
+To go along with effective administration and putting user security and wellbeing first, for a forem to be most useful, it maximizes its technical compatibility.
```

Comment:
```suggestion
To go along with effective administration and putting user security and wellbeing first, for a Forem to be most useful, it maximizes its technical compatibility.
```

Reply:
Agree with @jacobherrington's suggestions here and also want to add that we should use `well-being`, the standard noun format, rather than `wellbeing` here :)